packages/suite-native/android/app/build.gradle

```diff
@@ -117,10 +117,21 @@ android {
             include "armeabi-v7a", "x86", "arm64-v8a", "x86_64"
         }
     }
+    signingConfigs {
+        release {
+            // if (project.hasProperty('MYAPP_UPLOAD_STORE_FILE')) {
```

Comment:
Is this necessary?

Reply:
not sure. this comes from official documentation. so I would probably stick to it and uncommonent it. If nothing more, it will help us with updates as our code will look more or less same as the diff we use to determine necessary changes.